### PR TITLE
#134 App Settings Page complete

### PR DESCRIPTION
**Pull request by MackTr**
The appsettings page now has 3 buttons to provide interaction with the user. These buttons are privacy policy, colorblind theme, and google calendar integration.

### CODE REVIEW

File: SD-K9/src/app/pages/app-settings/app-settings.page.ts [hunk 1]
```diff
@@ -9,11 +10,27 @@ import { NavParams } from '@ionic/angular';
...
+
+  async changeTheme(){
+    this.themeSwitcher.cycleTheme();
+  }
+
+  async presentAlert() {
+    const alert = await this.alertController.create({
+      header: 'Alert',
+      subHeader: 'Subtitle',
+      message: 'It is NavConU\'s policy to respect your privacy regarding any information we may collect while operating our website. This Privacy Policy applies to www.NavConU.com (hereinafter, "us", "we", or "www.NavConU.com"). We respect your privacy and are committed to protecting personally identifiable information you may provide us through the Website. We have adopted this privacy policy ("Privacy Policy") to explain what information may be collected on our Website, how we use this information, and under what circumstances we may disclose the information to third parties. This Privacy Policy applies only to information we collect through the Website and does not apply to our collection of information from other sources.',
```
Comment (a-paggabao) [inline SD-K9/src/app/pages/app-settings/app-settings.page.ts:29]: Can you put the message in a config file?

Comment (MackTr) [reply SD-K9/src/app/pages/app-settings/app-settings.page.ts:29]: Yes that is a good idea, I'll look into it

File: SD-K9/src/app/pages/app-settings/app-settings.page.html [hunk 2]
```diff
@@ -12,16 +12,10 @@
...
   </ion-item>
   <ion-item>
-    <ion-label>Notifications</ion-label>
-  </ion-item>
-  <ion-item>
-    <ion-label>Privacy Policy</ion-label>
-  </ion-item>
-  <ion-item>
-    <ion-label>About</ion-label>
+    <ion-button (click)="changeTheme()">ColorBlind Theme</ion-button>
```
Comment (samiha-rahman) [inline SD-K9/src/app/pages/app-settings/app-settings.page.html:18]: Nice touch adding in the color-blind theme!

Comment (Weigiu) [reply SD-K9/src/app/pages/app-settings/app-settings.page.html:18]: Would it be possible to swap for a toggle instead ?

Comment (Weigiu) [inline SD-K9/src/app/pages/app-settings/app-settings.page.html:18]: Would it be possible to have a toggle instead to see when it is selected ?

Comment (MackTr) [reply SD-K9/src/app/pages/app-settings/app-settings.page.html:18]: Thanks for the feedback! Implemented!

**Pull request merged**